### PR TITLE
chore: add focus to textarea and change button open

### PR DESCRIPTION
**Pull request by emoriya**
This PR add focus on `textarea feedback` when email input isn't showing.

Add logic to show form top or bottom of `Button Trigger`

Add RadixUI Popover Primitive

### CODE REVIEW

File: src/Feedback/index.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ const Feedback = ({
   email,
   enableFeedbackText = true,
   enableEmoji = true,
+  openTop = false,
```
Comment (esoubihe) [inline src/Feedback/index.js:86]: You need to be aware that this should handle cases aside from the dashboard, so positioning elements based on a more generic prop is probably more appropriate.

File: src/Feedback/index.js [hunk 2]
```diff
@@ -235,20 +242,20 @@ const Feedback = ({
       ]}
       {...props}
     >
+      {side === "bottom" && (children
```
Comment (maxweldsouza) [inline src/Feedback/index.js:245]: You should use position absolute and transforms to correctly position the popover. Referring to open source Popover components will help.

**Comment by esoubihe** [comment]
We should actually use RadixUI's popover primitive here I think.

**Comment by emoriya** [comment]
> We should actually use RadixUI's popover primitive here I think.

I think it would be good, I'll refactor to use RadixUI's popover primitive

**Comment by emoriya** [comment]
Component is working now with radix popover.

**Review by maxweldsouza — COMMENTED** [review]
Looks good!

A small issue I noticed is that when you try to submit and empty form on firefox, the browser message "Please fill out this field" is positioned incorrectly. Here is a like to a video https://drive.google.com/file/d/1V6421nilnnTiW1yoCpnzxOmu_-Wc4XKi/view?usp=sharing

**Comment by emoriya** [comment]
> Looks good!
> 
> A small issue I noticed is that when you try to submit and empty form on firefox, the browser message "Please fill out this field" is positioned incorrectly. Here is a like to a video https://drive.google.com/file/d/1V6421nilnnTiW1yoCpnzxOmu_-Wc4XKi/view?usp=sharing

Fixed, it's a conflict with popover focus and focus function we have in code.

**Review by maxweldsouza — APPROVED** [review]

**Review by esoubihe — APPROVED** [review]
Nice work @emoriya!

**Pull request merged**